src/lib/util/misc.h

```diff
@@ -171,6 +171,14 @@ int8_t		fr_pointer_cmp(void const *a, void const *b);
 void		fr_quick_sort(void const *to_sort[], int min_idx, int max_idx, fr_cmp_t cmp);
 int		fr_digest_cmp(uint8_t const *a, uint8_t const *b, size_t length) CC_HINT(nonnull);
 
+int 		fr_file_touch(char const *filename, mode_t mode);
+
+/**  Remove a name from the filesystem.
+ */
+static inline int fr_file_remove(char const *filename) {
+	return remove(filename);
```

Comment:
We generally use unlink not remove, and this should set fr_strerror otherwise it's pretty pointless wrapping the function

Reply:
Should probably also check that it's a file and not a directory